src/Integration/NewConnectedMode/ConfigFileUtilities.cs

```diff
@@ -1,102 +0,0 @@
-/*
- * SonarLint for Visual Studio
- * Copyright (C) 2016-2018 SonarSource SA
- * mailto:info AT sonarsource DOT com
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this program; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- */
-
-using System;
-using System.Diagnostics;
-using System.IO;
-using Microsoft.Alm.Authentication;
-using Newtonsoft.Json;
-using SonarLint.VisualStudio.Integration.Helpers;
-using SonarLint.VisualStudio.Integration.Persistence;
-using SonarLint.VisualStudio.Integration.Resources;
-using SonarQube.Client.Helpers;
-
-namespace SonarLint.VisualStudio.Integration.NewConnectedMode
-{
-    internal static class ConfigFileUtilities
```

Comment:
No longer needed - all of this is in the new abstract class, along with the saving code.